### PR TITLE
chore: portalicious package fixes

### PR DESCRIPTION
**Pull request by aberonni**
Addressing PR comments from #5447 separately because I'd like to merge #5447 as-is

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] ~I do not need any deviation from our PR guidelines~ do not merge until #5447 has been merged

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,18 +13,22 @@
     "start:services": "cd services/ && docker compose -f docker-compose.yml -f docker-compose.development.yml up -d --build",
     "start:services:appmap": "cd services/ && docker compose -f docker-compose.yml -f docker-compose.development.yml -f docker-compose.development-appmap.yml up -d --build",
     "start:portal": "npm start --prefix interfaces/Portal/ -- --port 8888",
+    "start:portalicious": "npm start --prefix interfaces/Portalicious/ -- --port 8888",
```
Comment (elwinschmitz) [inline package.json:16]: I _love_ how this name doesn't help with "tab-command-completion" at all... 😑 😶

Comment (aberonni) [reply package.json:16]: I guess I could simultaneously rename the `portal` scripts to `old-portal`?

File: interfaces/Portalicious/src/index.html [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
-  <base href="/">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
-</head>
-<body>
-  <app-root></app-root>
-</body>
+  <head>
+    <meta charset="utf-8" />
+    <title>121 Portal(icious)</title>
```
Comment (elwinschmitz) [inline interfaces/Portalicious/src/index.html:5]: 🤔 I _see_ what you did there...

**Review by elwinschmitz — APPROVED** [review]

**Pull request merged**